lib/internal_metric/web.rb

```diff
@@ -112,6 +113,7 @@ def self.from_env_data(env, data, host)
       metric.crawler = !!data[:is_crawler]
       metric.logged_in = !!data[:has_auth_cookie]
       metric.background = !!data[:is_background]
```

Comment:
supernit: Was looking above why it's `background` vs `is_background`. I would prefer the `is` to indicate it's a boolean.

Reply:
Yeah the inconsistency is annoying here. But I think if we wanted to change it to be `metric.is_background`, we'd have to rename all the booleans on the Web metric

https://github.com/discourse/discourse-prometheus/blob/f524375e12b9eb8ed7d87974d171bce6b6fc8a7b/lib/internal_metric/web.rb#L21-L32

And I think refactoring all of those is probably beyond the scope of this PR.